src/jvm/test/org/jetbrains/kotlinx/lincheck_test/representation/RunConcurrentRepresentationTests.kt

```diff
@@ -386,7 +386,7 @@ class IncorrectHashmapRunConcurrentRepresentationTest : BaseRunConcurrentReprese
 class ThreadPoolRunConcurrentRepresentationTest : BaseRunConcurrentRepresentationTest<Unit>("run_concurrent_test/thread_pool/thread_pool") {
     // TODO: check why flakiness appears in this test
     override val isFlakyTest: Boolean
-        get() = true
+        get() = false
```

Comment:
Should we remove the override?

Reply:
If there will be no flakiness on CI anymore, then yes : )

Reply:
Unfortunately, the test is still flaky due to accidental changes in the object numeration:
https://github.com/JetBrains/lincheck/issues/593

We should investigate and fix it separately.